plugins/importexport/crossref/CrossRefExportPlugin.inc.php

```diff
@@ -296,12 +301,25 @@ function depositXML($objects, $context, $filename) {
 					]
 				]
 			);
-		} catch (GuzzleHttp\Exception\RequestException $e) {
-			if ($e->getResponse()->getStatusCode() != CROSSREF_API_DEPOSIT_ERROR_FROM_CROSSREF) {
-				return [['plugins.importexport.common.register.error.mdsError', 'No response from server.']];
+		 } catch (GuzzleHttp\Exception\RequestException $e) {
+			$returnMessage = $e->getMessage();
+			if ($e->hasResponse()) {
+				$eResponseBody = $e->getResponse()->getBody(true);
+				$eStatusCode = $e->getResponse()->getStatusCode();
+				if ($eStatusCode == CROSSREF_API_DEPOSIT_ERROR_FROM_CROSSREF) {
+					$xmlDoc = new DOMDocument();
+					$xmlDoc->loadXML($eResponseBody);
+					$batchIdNode = $xmlDoc->getElementsByTagName('batch_id')->item(0);
+					$msg = $xmlDoc->getElementsByTagName('msg')->item(0)->nodeValue;
+					$status = CROSSREF_STATUS_FAILED;
+					$this->updateDepositStatus($context, $objects, $status, $batchIdNode->nodeValue, $msg);
```

Comment:
Would it be worth saving all of `$eResponseBody` here? As mentioned in https://github.com/pkp/pkp-lib/issues/6887#issuecomment-810814396, the full XML response contains other low-level info and statistic information that would be useful to retain, especially for debugging purposes.